packages/salesforcedx-vscode-lightning/esbuild.config.mjs

```diff
@@ -6,10 +6,64 @@
  */
 import { nodeConfig } from '../../scripts/bundling/node.mjs';
 import { build } from 'esbuild';
+import { copyFileSync, mkdirSync, existsSync } from 'fs';
+import { join, dirname } from 'path';
 
 await build({
   ...nodeConfig,
-  external: ['vscode', 'applicationinsights', '@salesforce/lightning-lsp-common', '@salesforce/aura-language-server'],
+  external: ['vscode', 'applicationinsights', '@salesforce/lightning-lsp-common'],
```

Comment:
I think the current version of nodeConfig has an `external` property (just `vscode` for now)

You can spread that and then add what you need.  Check if `applicationinsights` is necessary...most pkgs don't have to do that.